### PR TITLE
STM documentations and thingy fix

### PR DESCRIPTION
**Pull request by malw4**

### CODE REVIEW

File: docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md [hunk 1]
```diff
@@ -6,18 +6,29 @@ Integrate your B-L462E-CELL1 Discovery kit board along with the TYPE 1SE module
...
 - A user with access to the Coiote IoT Device Management platform.
 
-## Step 1: Cloning the Anjay freeRTOS client repository
+
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-freertos-client](https://github.com/AVSystem/Anjay-freertos-client/releases/).
+
+In this case we are going to use the Anjay-freertos-client-B-L462E-CELL1-TYPE1SC.bin file which after downloaded must be added to B-L462E-CELL1/TYPE1SC board folder.
```
Comment (JZimnol) [inline docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md:18]:
```suggestion
In this case we are going to use the Anjay-freertos-client-B-L462E-CELL1-TYPE1SC.bin file which after being downloaded must be added to B-L462E-CELL1/TYPE1SC board folder.
```

File: docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md [hunk 2]
```diff
@@ -56,10 +57,31 @@ To connect the board:
...
+         - In the **Security mode** section, select the **PSK (Pre-Shared Key)** mode:
               - In the **Key identity** field, type `test_device`
               - In the **Key** field, type the shared secret used in the device-server authentication.
     4. Click the **Add device** button and **Confirm** in the confirmation pop-up.
     5. In the **Connect your device** step, wait for the board to connect.
     6. Click **Go to device** to see your added device dashboard.
-        ![Registered device](images/registered_device.png "Registered device")
+
+    !!! tip
+        LwM2M Server URI, endpoint name and other informations you can read from **configuration** tab.
```
Comment (JZimnol) [inline docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md:68]:
```suggestion
        LwM2M Server URI, endpoint name and other informations can be found in the **configuration** tab.
```

File: docs/Anjay_integrations/Anjay_ST_integration/B-L475E-IOT01A.md [hunk 3]
```diff
@@ -5,22 +5,33 @@ Integrate your B-L475E-IOT01A Discovery kit board.
...
+- Installed **ST-Link** or **OpenOCD** debugger.
 - A user with access to the Coiote IoT Device Management platform and appropriate permissions.
 
-## Step 1: Getting Zephyr and Python dependencies
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client/releases).
+
+In this case we are going to use the demo_B-L475E-IOT01A1_merged.bin file which after downloaded must be added to B-L475E-IOT01A board folder.
```
Comment (JZimnol) [inline docs/Anjay_integrations/Anjay_ST_integration/B-L475E-IOT01A.md:17]:
```suggestion
In this case we are going to use the demo_B-L475E-IOT01A1_merged.bin file which after being downloaded must be added to B-L475E-IOT01A board folder.
```

File: docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCO-MONARCH.md [hunk 4]
```diff
@@ -6,18 +6,28 @@ Integrate your P-L496G-CELL02 Discovery kit board along with the Sequans' Monarc
...
+- Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
 - A user with access to the Coiote IoT Device Management platform.
 
-## Step 1: Cloning the Anjay freeRTOS client repository
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-freertos-client](https://github.com/AVSystem/Anjay-freertos-client/releases/).
+
+In this case we are going to use the Anjay-freertos-client-STM32L496G-MONARCH.bin file which after downloaded must be added to STM32L496G-DISCO/MONARCH board folder.
```
Comment (JZimnol) [inline docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCO-MONARCH.md:17]:
```suggestion
In this case we are going to use the Anjay-freertos-client-STM32L496G-MONARCH.bin file which after being downloaded must be added to STM32L496G-DISCO/MONARCH board folder.
```

File: docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCO-MONARCH.md [hunk 5]
```diff
@@ -56,10 +57,31 @@ To connect the board:
...
-         - In the **Security mode** section, select the **PSK** mode:
+         - In the **Security mode** section, select the **PSK (Pre-shared Key)** mode:
               - In the **Key identity** field, type `test_device`
               - In the **Key** field, type the shared secret used in the device-server authentication.
     4. Click the **Add device** button and **Confirm** in the confirmation pop-up.
     5. In the **Connect your device** step, wait for the board to connect.
     6. Click **Go to device** to see your added device dashboard.
+
+    !!! tip
+        LwM2M Server URI, endpoint name and other informations you can read from **configuration** tab.
```
Comment (JZimnol) [inline docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCO-MONARCH.md:68]:
```suggestion
        LwM2M Server URI, endpoint name and other informations you can be found in the **configuration** tab.
```

File: docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCOBG96.md [hunk 6]
```diff
@@ -6,18 +6,28 @@ Integrate your P-L496G-CELL02 Discovery kit board along with the default-provide
...
+- Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
 - A user with access to the Coiote IoT Device Management platform.
 
-## Step 1: Cloning the Anjay freeRTOS client repository
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-freertos-client](https://github.com/AVSystem/Anjay-freertos-client/releases/).
+
+In this case we are going to use the Anjay-freertos-client-STM32L496G-BG96.bin file which after downloaded must be added to STM32L496G-DISCO/BG96 board folder.
```
Comment (JZimnol) [inline docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCOBG96.md:17]:
```suggestion
In this case we are going to use the Anjay-freertos-client-STM32L496G-BG96.bin file which after being downloaded must be added to STM32L496G-DISCO/BG96 board folder.
```

File: docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCOBG96.md [hunk 7]
```diff
@@ -59,10 +59,32 @@ To connect the board:
...
+         - In the **Security mode** section, select the **PSK (Pre-Shared Key)** mode:
               - In the **Key identity** field, type `test_device`
               - In the **Key** field, type the shared secret used in the device-server authentication.
     4. Click the **Add device** button and **Confirm** in the confirmation pop-up.
     5. In the **Connect your device** step, wait for the board to connect.
     6. Click **Go to device** to see your added device dashboard.
-        ![Registered device](images/registered_device.png "Registered device")
+
+    !!! tip
+        LwM2M Server URI, endpoint name and other informations you can read from **configuration** tab.
```
Comment (JZimnol) [inline docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCOBG96.md:70]:
```suggestion
        LwM2M Server URI, endpoint name and other informations you can be found in the **configuration** tab.
```

**Review by JZimnol — COMMENTED** [review]
Added some comments and tips.  
Also, do you really want to modify the `Thingy91.md` and `M5StickC.md` files in this pull request?

**Comment by mkrawczyk-avs** [comment]
merge because of deadline

File: docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md [hunk 8]
```diff
@@ -6,18 +6,29 @@ Integrate your B-L462E-CELL1 Discovery kit board along with the TYPE 1SE module
...
 - A user with access to the Coiote IoT Device Management platform.
 
-## Step 1: Cloning the Anjay freeRTOS client repository
+
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-freertos-client](https://github.com/AVSystem/Anjay-freertos-client/releases/).
+
+In this case we are going to use the Anjay-freertos-client-B-L462E-CELL1-TYPE1SC.bin file which after being downloaded must be added to B-L462E-CELL1/TYPE1SC board folder.
```
Comment (Mierunski) [inline docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md:18]: `which after being downloaded must be added to B-L462E-CELL1/TYPE1SC board folder.` what do you mean here?

File: docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md [hunk 9]
```diff
@@ -6,18 +6,29 @@ Integrate your B-L462E-CELL1 Discovery kit board along with the TYPE 1SE module
...
+#### Step 1: Cloning the Anjay freeRTOS client repository
 
 Enter the command line interface on your machine and paste the following command:
 
    ```
    git clone --recursive https://github.com/AVSystem/Anjay-freertos-client
    ```
 
-## Step 2: Compiling the board
+#### Step 2: Compiling the board
```
Comment (Mierunski) [inline docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md:31]: `Compiling the application`

File: docs/Anjay_integrations/Anjay_ST_integration/B-L475E-IOT01A.md [hunk 10]
```diff
@@ -5,22 +5,33 @@ Integrate your B-L475E-IOT01A Discovery kit board.
...
+- Installed **ST-Link** or **OpenOCD** debugger.
 - A user with access to the Coiote IoT Device Management platform and appropriate permissions.
 
-## Step 1: Getting Zephyr and Python dependencies
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client/releases).
+
+In this case we are going to use the demo_B-L475E-IOT01A1_merged.bin file which after being downloaded must be added to B-L475E-IOT01A board folder.
```
Comment (Mierunski) [inline docs/Anjay_integrations/Anjay_ST_integration/B-L475E-IOT01A.md:17]: `which after being downloaded must be added to B-L475E-IOT01A board folder.` same, what does it mean?

File: docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCO-MONARCH.md [hunk 11]
```diff
@@ -6,18 +6,28 @@ Integrate your P-L496G-CELL02 Discovery kit board along with the Sequans' Monarc
...
+- Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
 - A user with access to the Coiote IoT Device Management platform.
 
-## Step 1: Cloning the Anjay freeRTOS client repository
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-freertos-client](https://github.com/AVSystem/Anjay-freertos-client/releases/).
+
+In this case we are going to use the Anjay-freertos-client-STM32L496G-MONARCH.bin file which after being downloaded must be added to STM32L496G-DISCO/MONARCH board folder.
```
Comment (Mierunski) [inline docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCO-MONARCH.md:17]: and here

File: docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCOBG96.md [hunk 12]
```diff
@@ -6,18 +6,28 @@ Integrate your P-L496G-CELL02 Discovery kit board along with the default-provide
...
+- Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
 - A user with access to the Coiote IoT Device Management platform.
 
-## Step 1: Cloning the Anjay freeRTOS client repository
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-freertos-client](https://github.com/AVSystem/Anjay-freertos-client/releases/).
+
+In this case we are going to use the Anjay-freertos-client-STM32L496G-BG96.bin file which after being downloaded must be added to STM32L496G-DISCO/BG96 board folder.
```
Comment (Mierunski) [inline docs/Anjay_integrations/Anjay_ST_integration/STM32L496G-DISCOBG96.md:17]: and here

**Review by Mierunski — COMMENTED** [review]
Need to address it in https://github.com/AVSystem/developer-zone/pull/66

**Pull request merged**